packages/pwa-buildpack/lib/BuildBus/mapHooksToTargets.js

```diff
@@ -33,6 +33,7 @@ const VALID_TYPES = new Map();
  * Dictionary of Tapable Hook classes to expose under these new names.
  * @public
  * @type {Object.<string,Tapable.Hook>}
+ * @see [Tapable]{@link https://github.com/webpack/tapable}
```

Comment:
You know what? I don't think this file or these functions should be public. They're not organized along with the rest of the Buildpack API, so they `mapHooksToTargets` looks weird in the navigation.

I shouldn't have set the @public keywords on these functions. They're utility functions that BuildBus uses. Sorry about that.

Reply:
topic file deleted